src/main/java/com/example/javaOjt/services/AuthorService.java

```diff
@@ -109,7 +109,20 @@ public GetAuthorResponse patchAuthor(Integer id, String name) {
   }
 
   public GetAuthorResponse deleteAuthor(Integer id, Boolean permanent) {
```

Comment:
service 層で `@Transactional` をつけましょう！

```suggestion
  @Transactional
  public GetAuthorResponse deleteAuthor(Integer id, Boolean permanent) {
```